api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -51,8 +51,166 @@ service OktaService {
   rpc DeleteOktaAssignment(DeleteOktaAssignmentRequest) returns (google.protobuf.Empty);
   // DeleteAllOktaAssignments removes all Okta assignments.
   rpc DeleteAllOktaAssignments(DeleteAllOktaAssignmentsRequest) returns (google.protobuf.Empty);
+
+  // ValidateClientCredentials checks if the provided client credentials are valid.
+  rpc ValidateClientCredentials(ValidateClientCredentialsRequest) returns (ValidateClientCredentialsResponse);
+  // EnrollIntegration enrolls a new integration with the specified parameters.
+  rpc CreateIntegration(CreateIntegrationRequest) returns (CreateIntegrationResponse);
+  // UpdateIntegration updates the settings or properties of an existing integration.
+  rpc UpdateIntegration(UpdateIntegrationRequest) returns (UpdateIntegrationResponse);
+  // GetApps retrieves a list of apps based on specified filter criteria.
+  rpc GetApps(GetAppsRequest) returns (GetAppsResponse);
+  // GetGroups retrieves a list of groups based on specified filter criteria.
+  rpc GetGroups(GetGroupsRequest) returns (GetGroupsResponse);
```

Comment:
Still not clear by the godoc where this is retrieving apps/groups from, if it's from Okta, do you mind just updating the comments to say "retrieves a list of apps from Okta based on..." and same for groups?